Makefile

```diff
@@ -63,3 +68,19 @@ update-lambda-dev: ## Updates the lambda with whatever is the most recent image
 		--function-name $(FUNCTION_DEV) \
 		--image-uri $(ECR_URL_DEV):latest
 		
+## Terraform-generated Makefile developer Deploy Commands ###
+dist-stage: ## Build docker container (intended for developer-based manual build)
```

Comment:
These aren't really intended for manual developer build/publish, right?

Reply:
The `stage` commands are in there temporarily until we resolve our conversation in InfraEng on Wednesday regarding Terraform infra repos and branches (right now, dev is a separate repo for app infrastructure repos). This will make a lot more sense after Wednesday morning!